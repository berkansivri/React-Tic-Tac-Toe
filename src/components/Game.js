import React from 'react'
import Board from './Board'

const Game = () => {
  return (
    <div className="game">
      <div className="game-board">
        <Board />
      </div>
      <div className="game-info">
        {/* TODO */}
      </div>
    </div>
  )
}

export { Game as default }