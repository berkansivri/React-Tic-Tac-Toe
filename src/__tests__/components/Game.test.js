import React from 'react'
import { shallow } from 'enzyme'
import Game from '../../components/Game'

test("should render game component correctly", () => {
  let wrapper = shallow(<Game />)
  expect(wrapper).toMatchSnapshot()
})
