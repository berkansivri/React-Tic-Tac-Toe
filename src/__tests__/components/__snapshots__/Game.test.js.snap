// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render game component correctly 1`] = `
<ContextProvider
  value={
    Object {
      "dispatch": [Function],
      "moves": Array [],
      "setTurn": [Function],
      "turn": "X",
    }
  }
>
  <div
    className="game"
  >
    <div
      className="game-board"
    >
      <Board />
    </div>
    <div
      className="game-info"
    >
      <Moves />
    </div>
  </div>
</ContextProvider>
`;
